cli/compose/convert/service_test.go

```diff
@@ -623,3 +624,48 @@ func TestConvertUpdateConfigParallelism(t *testing.T) {
 	})
 	assert.Check(t, is.Equal(parallel, updateConfig.Parallelism))
 }
+
+func TestConvertServiceCapAddAndCapDrop(t *testing.T) {
+	service := composetypes.ServiceConfig{
```

Comment:
Maybe an additional test for no cap's set?

Reply:
Included